src/data/ACTIONS/root/BLU.ts

```diff
@@ -1008,4 +1011,196 @@ export const BLU = ensureActions({
 		statusesApplied: ['BLEEDING'],
 		damageType: MAGICAL,
 	},
+	GOBLIN_PUNCH: {
+		id: 34563,
+		name: 'Goblin Punch',
+		icon: 'https://xivapi.com/i/003000/003358.png',
+		onGcd: true,
+		gcdRecast: 2500,
```

Comment:
Should we go ahead and tag the GCDs (or otherwise? BLU's weird) with the speedAttribute: Attributes.SPELL_SPEED property to make sure the timeline display respects the player's speed?

Reply:
I think this is a good idea, though I'd rather do it in a separate PR since I'll need to go and check every spell, since SOME of our oGCDs are also affected by SpS.

(also, to be honest... I play with zero SpS both as BLU and RDM so I didn't even realize this was an option)

Reply:
Well, that actually only took a couple of minutes to verify in-game, I didn't remember you could just see the tooltip of spells in the spell book without slotting them into your actions.  PR now includes `speedAttribute: Attributes.SPELL_SPEED` for everything that's affected, including three weird edge cases (White Death and Divine Cataract are affected, Phantom Flurry's kick is not)